kaggle_environments/core.py

```diff
@@ -84,6 +85,16 @@ def make(environment, configuration={}, steps=[], debug=False):
     raise InvalidArgument("Unknown Environment Specification")
 
 
+def act_agent(args):
```

Comment:
Only top level methods are picklable so I moved this method to the top so we could parallelize it easily.